core/src/main/java/io/kestra/core/tasks/storages/LocalFiles.java

```diff
@@ -40,9 +40,54 @@
 @NoArgsConstructor
 @Schema(
     title = "Allow to create files in the local filesystem or to send files from the local filesystem to the internal storage.",
-    description = "This task should be used with the WorkingDirectory task to be able to access the same local filesystem within multiple tasks."
+    description = "This task should be used with the WorkingDirectory task to be able to access the same local filesystem within multiple tasks. Note that this is a flowable task, so setting `disabled: true` will not work on this task."
```

Comment:
This would be nice to add it to every flowable to keep consistency (and not discovery information about a task by digging in another)

Reply:
is there a list showing which ones are flowable? Are all Core tasks except for Scripts and Debugs Flowable?

Reply:
@Skraye I cross-checked, and the info I got somewhere that flowable tasks cannot be disabled is not true. I validated this on several task types, incl. WorkingDirectory, AllowFailure, Flow, EachParallel, Dag, and they all can be disabled. 

This LocalFiles task is an outlier so I think we can merge this as is and update later if some other tasks cannot be disabled.

Reply:
Good for me, but then maybe dont say "this is a flowable task, so setting ..." ?